engine/src/flutter/lib/web_ui/lib/src/engine/canvaskit/text.dart

```diff
@@ -87,6 +89,12 @@ class CkParagraphStyle implements ui.ParagraphStyle {
       skTextStyle.fontStyle = toSkFontStyle(fontWeight, fontStyle);
     }
 
+    final int weightValue = fontWeight?.value ?? ui.FontWeight.normal.value;
```

Comment:
We want to avoid adding `fontVariations` to the text style when we can because it requires us to malloc an array on the WASM side.

Reply:
Unfortunately I don't know of a way to work around this.

Some variable fonts may not have the default value of the `wght` attribute configured to match the value of `FontWeight.normal`.  If the engine does not explicitly set the `wght` value in `fontVariations`, then these fonts will not render as intended at the normal weight.

Reply:
That makes sense. I guess we can live with it, given that it seems that browsers apparently always set axes according to CSS, it makes sense for us to set them according to Flutter text style properties (source: https://groups.google.com/g/skia-discuss/c/eFFDObvJyQ8)